src/main/java/Duke.java

```diff
@@ -1,10 +1,186 @@
+import java.util.Scanner;
+
 public class Duke {
```

Comment:
Perhaps you can change the filename and class name to `Lambo` to match the name you have given your chatbot!